src/mantle/framework/manifest/class-model-manifest.php

```diff
@@ -24,7 +24,7 @@ class Model_Manifest {
 	/**
 	 * Manifest from the disk.
 	 *
-	 * @var array
+	 * @var array|null
```

Comment:
The base value for an array is `[]`.

[_source_](https://github.com/szepeviktor/very-clean-code/blob/master/README.md#-base-value-of-an-array)

Reply:
🎈 Fun time!

2 elements `['one', 'two']`
1 element `['one']`
base value `[]`
-1 elements `null` 😲